src/components/modals/AddMovieModal/AddMovieModal.js

```diff
@@ -1,62 +1,98 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
 import styles from './AddMovieModal.module.scss';
 import ModalBase from '../ModalBase/ModalBase';
 import { GENRES } from '~/services/mock-data';
 import FormField from '../../FormField/FormField';
 import FormFieldSelect from '../../FormField/FormFieldSelect';
 import { MoviePropTypes } from '~/utils/CommonPropTypes';
+import { sortAbc } from '~/utils/sorting';
 
 const FIELDS = [
   {
-    label: 'Movie Id', type: 'text', readOnly: true, key: 'id',
+    label: 'Movie Id',
+    type: 'text',
+    readOnly: true,
+    key: 'id',
   },
   { label: 'Title', type: 'text', key: 'title' },
-  { label: 'Release Date', type: 'date' },
-  { label: 'Movie Url', type: 'text' },
+  { label: 'Release Date', type: 'date', key: 'release_date' },
+  { label: 'Poster Url', type: 'text', key: 'poster_path' },
   {
-    label: 'Genre', type: 'select', values: GENRES, key: 'genres',
+    label: 'Genre',
+    type: 'select',
+    values: GENRES,
+    key: 'genres',
   },
-  { label: 'Overview', type: 'textarea' },
-  { label: 'Runtime', type: 'textarea' },
+  { label: 'Runtime', type: 'number', key: 'runtime' },
+  { label: 'Overview', type: 'textarea', key: 'overview' },
 ];
 
-const GENRES_OPTIONS = GENRES.map((genre) => ({
-  value: genre.label,
-  label: genre.label,
-}));
+const mapOptions = (options) => (options ? options.map((option) => ({ label: option, value: option })) : []);
```

Comment:
Personally I’m not sure that `mapOptions` suite this function. Maybe `getOptions` will be better?

Reply:
Fixed